docs/Rakefile

```diff
@@ -19,19 +19,23 @@ task test: :build do
     "./_site",
     parallel: { in_threads: 4 },
     favicon: true,
-    http_status_ignore: [0, 403],
-    assume_extension: true,
+    ignore_status_codes: [0, 403],
     check_favicon: true,
     check_opengraph: true,
     check_html: true,
     check_img_http: true,
     enforce_https: true,
-    file_ignore: [
+    ignore_files: [
       %r{Kickstarter-Supporters},
     ],
-    url_ignore: [
-      "http://formulae.brew.sh",
-      %r{https://github.com/Homebrew/brew/edit/gh-pages//},
-    ]
+    ignore_urls: [
+      %r{https://formulae.brew.sh"},
+      %r{https://github.com/}
+    ],
+    cache: {
+      timeframe: {
+        external: "1h"
```

Comment:
@issyl0 does this setup the GitHub Actions cache across workflows? If not, might be a nice extension.